### ISSUE TITLE
Zig fmt multi-line String

### ISSUE THREAD

**Issue by amrojjeh** [ziglang/zig#8440]
```zig
pub fn help() void {
    std.debug.print(
        \\Welcome to the To-Do List!
        \\=== Examples ===
        \\todo Literature Review : Adds lit review as a task
        \\todo Literature Review due Wednesday: Adds lit review as a task which is due Wednesday
        , .{});
}
````
```
D:\Projects\Zig-to-do-list\src>zig fmt main.zig
main.zig:27:37: error: expected ',', found invalid bytes
        \\Welcome to the To-Do List!
                                    ~
main.zig:27:37: error: expected ')', found invalid bytes
        \\Welcome to the To-Do List!
                                    ~
```

The code compiles and works, yet `zig fmt` seems to struggle with formatting the file for some odd reason. Is this a bug?

**Comment by amrojjeh** [comment]
Apparently, this had to do with the fact that I was using CRLF. Converting it to LF fixed the issue.

**Comment by pfgithub** [comment]
zig fmt should be able to automatically resolve CRLF errors, so this is probably still an issue

**Comment by Mouvedia** [comment]
@pfgithub it should have been fixed by 4f43a4b.
